packages/common/src/intl/locales/en/distribution.json

```diff
@@ -27,15 +27,15 @@
   "label.select-rows-to-delete-them": "Select rows to delete them",
   "label.units-in-pack-size-of_one": "unit in pack size of",
   "label.units-in-pack-size-of_other": "units in pack size of",
-  "message.cant-delete-invoices": "Can only delete invoices with a status of 'New' or 'Allocated'",
-  "message.cant-delete-requisitions": "Can only delete requisitions with a status of 'Draft'",
-  "message.deleted-invoices": "Deleted {{number}} invoices",
-  "message.deleted-requisitions": "Deleted {{number}} requisitions",
-  "message.select-rows-to-delete": "Select rows to delete",
+  "messages.cant-delete-invoices": "Can only delete invoices with a status of 'New' or 'Allocated'",
```

Comment:
yeah, sorry. I wandered offroad with this one.. noticed that some were `message` and some `messages` so I standardised.

Reply:
haha all good, prefer to get em done even if it bloats the diff a bit!